factory/glideFactoryMonitorAggregator.py

```diff
@@ -520,7 +425,7 @@ def aggregateStatus(in_downtime):
                     a_el = int(tp_el[a])
                     val_dict[f"{tp_str}{a}"] = a_el
         glideFactoryMonitoring.monitoringConfig.write_rrd_multi(
-            "total/%s/Status_Attributes" % ("frontend_" + fe), "GAUGE", updated, val_dict
+            os.path.join("total", "frontend_" + fe, "Status_Attributes"), "GAUGE", updated, val_dict
```

Comment:
Would it be better to change `"frontend_" + fe` to `f"frontend_{fe}"`?

Reply:
done